__tests__/Auth0Client/getTokenSilently.test.ts

```diff
@@ -663,6 +664,68 @@ describe('Auth0Client', () => {
       );
     });
 
+    describe('concurrency', () => {
+      it('should call _getTokenSilently multiple times when no call in flight concurrently', async () => {
+        const client1 = setup();
```

Comment:
Just for readability in these tests, I'd avoid calling this `client1` if there is only one client. I was expecting to see a second one here, especially given that it's a test about concurrency.

Reply:
Again, copy paste issue from the test below. Will fix it